daemon/containerd/image.go

```diff
@@ -23,6 +23,7 @@ import (
 
 var shortID = regexp.MustCompile(`^([a-f0-9]{4,64})$`)
 
+// GetContainerdImage returns the containerd image corresponding to the image referred to by refOrID
```

Comment:
🙈 should end with a full-stop

```suggestion
// GetContainerdImage returns the containerd image corresponding to the image referred to by refOrID.
```

Reply:
Should we mention that `platform` is currently ignored?